sphinx/testing/fixtures.py

```diff
@@ -21,6 +21,21 @@
 from sphinx.testing.util import SphinxTestApp, SphinxTestAppWrapperForSkipBuilding
 
 
+markers = [
```

Comment:
How about renaming this like `DEFAULT_MARKERS` or something similar? I feel it would be better to use a CONSTANT name and good adjective for this.

Reply:
@tk0miya maybe `PUBLIC_MARKERS`?

Reply:
@tk0miya however, maybe `PUBLIC_MARKERS` is unsuitable since it suggests some "private" markers also exist somewhere hidden.

Reply:
`DEFAULT_INSTALLED_MARKERS` or `DEFAULT_ENABLED_MARKERS`?

Reply:
Changed to `DEFAULT_ENABLED_MARKERS`